src/layer/AbstractLayer.ts

```diff
@@ -201,19 +243,29 @@ export class AbstractLayer {
     bbox: BBox,
     fromTime: Date,
     toTime: Date,
-    maxFindTilesRequests: number = 50,
-    tilesPerRequest: number = 50,
+    maxFindTilesRequests: number | null = 50,
```

Comment:
Why is this needed? Isn't it already assigned a value if not passed?

Reply:
I assume because you wanted to pass `overrideOrbitTimeMinutes` but didn't want to use `undefined`.

Reply:
Yes - in other places this library uses `null` as marker that some parameter was not specified and should be set to the default value. I am not sure if this was the best possible decision, but I used the same convention here for consistency.

Does that sound like a good idea or do you think a different approach would be better?

Reply:
I think this is an example why it's better to have a params object than separate params.

Reply:
Yes, but then you _must_ name all the parameters, every time. Unfortunately JS doesn't have a real "named params" mechanism like Python does.

Reply:
When calling, you need to name the parameters:
```
  myFunction({ parameterA: 15, parameterB: 30 });
```

You can't just say:
```
  myFunction({15, 50});  // invalid syntax
```

Anyway... this is probably a discussion for the chat. :)